### PR TITLE
skeletal aws_iam_ policies resource

### PR DESCRIPTION
**Pull request by rx294**

### CODE REVIEW

File: docs/resources/aws_iam_policys.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: About the aws_iam_policys Resource
+---
+
+# aws_iam_policys
+
+Use the `aws_iam_policys` InSpec audit resource to test properties of some or all AWS IAM Policys.
+
+To test properties of all managed policies that are available in your AWS account, including your own customer-defined managed policies and all AWS managed policies.
```
Comment (clintoncwolfe) [inline docs/resources/aws_iam_policys.md]: Please prefix with "It may be used to test...." to form a complete sentence.

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Looks good - just a couple of docs changes.  Thanks for another excellent contribution!

**Comment by rx294** [comment]
ah my bad....i have made the corrections thanks @clintoncwolfe

**Review by clintoncwolfe — APPROVED** [review]

**Comment by rx294** [comment]
Thank you Clinton!

**Pull request merged**